Configuration/Eras/python/Era_Run2_2017_HCALdev_cff.py

```diff
@@ -0,0 +1,11 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2016_cff import Run2_2016
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
+from Configuration.Eras.Modifier_run2_HE_2017_cff import run2_HE_2017
+from Configuration.Eras.Modifier_run2_HF_2017_cff import run2_HF_2017
+from Configuration.Eras.Modifier_run2_HCAL_2017_cff import run2_HCAL_2017
+
+Run2_2017_HCALdev = cms.ModifierChain(Run2_2016, phase1Pixel, trackingPhase1, run2_HE_2017, run2_HF_2017, run2_HCAL_2017)
```

Comment:
@kpedro88 (and @mmusich as perhaps this impacts also the GT)

- two comments
1) I believe we agreed to have the new pixel config with the hcaldev scenario. This has the old one.
2) It seems we have a divergence in the stack of modifiers here and in Era_Run2_2017_NewFPix_cff.py which is  simply

Run2_2017_NewFPix = cms.ModifierChain(Run2_2017, phase1PixelNewFPix)